sentry_sdk/client.py

```diff
@@ -482,6 +482,9 @@ def _prepare_event(
                     "event_processor", data_category="span", quantity=spans_delta
                 )
 
+        # TODO: Where should I put this?
+        event["contexts"]["flags"] = scope._flag_manager.serialize()
```

Comment:
@antonpirker Could you help me fill in the gaps here?  Where should this live?  Also are scopes nested?  Would it be possible to merge the flags from a global scope and a thread scope here?